src/containers/IntentService.jsx

```diff
@@ -103,23 +119,45 @@ export default class IntentService extends Component {
           <p>{t(error.message)}</p>
           <p>{t('intent.service.error.cause', {error: error.reason})}</p>
         </div>}
-        { !isFetching && !error && konnector &&
-          <div className='coz-service-layout'>
-            <ServiceBar
-              appEditor={data.cozyAppEditor}
-              appName={data.cozyAppName}
-              iconPath={`../${data.cozyIconPath}`}
-              onCancel={() => this.cancel()}
-              {...this.context}
-             />
+        <div className='coz-service-layout'>
+          <ServiceBar
+            appEditor={data.cozyAppEditor}
+            appName={data.cozyAppName}
+            iconPath={`../${data.cozyIconPath}`}
+            onCancel={() => this.cancel()}
+            {...this.context}
+           />
+          {
+            !isFetching && !error && konnector &&
             <CreateAccountService
               konnector={konnector}
               onCancel={() => this.cancel()}
               onSuccess={account => this.terminate(account)}
               disableSuccessTimeout={disableSuccessTimeout}
               {...this.context}
               />
-          </div>}
+          }
+          {
+            !isFetching && !error && !konnector && konnectors &&
+            <ul className='connector-list'>
```

Comment:
I feel that it could permit a code a little bit simpler if we handle the part "one konnector only" and "a list of konnectors" in two sub-components. What do you think ?